disabled_appveyor.yml

```diff
@@ -0,0 +1,81 @@
+# This file was automatically generated by conda-smithy. To update a component of this
```

Comment:
For `ptyprocess` this looks weird as there won't be a Windows build. @jakirkham decided to keep the AppVeyor yaml configuration in the feedstock as a place holder for those who want to try it in another project. It is harmless to keep that file around.

Note that `conda-smithy` does not really need that file as a new `appveyor.yml` will be generated if the Windows skip is removed.